### DISCUSSION TITLE
[FR] 已经存在的时间戳批量支持视频网站原生时间戳

### DISCUSSION THREAD

**Discussion by bcdax110** [aidenlx/media-extended#307] [Q&A] [answered]
#284 提到的网站原生时间戳在新添加时间戳时会自带，但是在此之前添加的时间戳并不会附带网站原生时间戳。

希望可以添加一个命令，用来转换当前文件中那些没有添加网站原生时间的时间戳

**Comment by aidenlx** [comment]
这个没计划支持，你看下批量替换能不能解决

**Comment by bcdax110** [comment, marked as answer]
好的，用`\[(.+?)\]\(https://www\.bilibili\.com/video/BV(\w+?)\?p=(\d+?)#t=(.+?)\)`规则替换成`[$1](https://www.bilibili.com/video/BV$2?p=$3&t=$4#t=$4)`似乎可以解决
